docs/man_pages/project/testing/debug-android.md

```diff
@@ -35,6 +35,11 @@ Debugs your project on a connected device, in a native emulator or in Genymotion
 
 * You must have Chrome installed on your system.
 
+<% if(isMacOS) { %>
```

Comment:
This tag won't work in the current isHtml block as the OS tags work in the console. My advice is to go with a `<br/>` right next to the period above. In this case, however, you would need to make this "If you are using a non-standard named Chrome on an OS X system (for example, a nightly Canary update), you need to set this name in the...

Also, please apply code formatting to the setting name.

```
`this is how you do code formatting`
```